### PR TITLE
Run3-gex160X Update the overlap finding scripts for Run3/Run2 scenarios

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Update the overlap finding scripts for Run3/Run2 scenarios

#### PR validation:

Use the scripts to estimate overlaps in Run3/Run2 scenarios

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by bsunanda** [comment]
@srimanob Please approve this

**Comment by srimanob** [comment]
+Upgrade

File: SimG4Core/PrintGeomInfo/test/python/g4OverlapCheckDDD_cfg.py [hunk 1]
```diff
@@ -32,25 +32,27 @@
...
 
+baseName = "cmsDDD" + options.geometry
+geomName = "Configuration.Geometry.GeometryExtended" + options.geometry + "Reco_cff"
+
 if (options.geometry == "2016"):
     from Configuration.Eras.Era_Run2_2016_cff import Run2_2016
     process = cms.Process('G4PrintGeometry',Run2_2016)
-    process.load('Configuration.Geometry.GeometryExtended2016Reco_cff')
-    baseName = 'cms2016'
+elif (options.geometry == "2018"):
```
Comment (perrotta) [inline SimG4Core/PrintGeomInfo/test/python/g4OverlapCheckDDD_cfg.py:41]:
```suggestion
elif (options.geometry == "2017"):
```

File: SimG4Core/PrintGeomInfo/test/python/g4OverlapCheckDD4hep_cfg.py [hunk 2]
```diff
@@ -17,9 +23,28 @@
 options.parseArguments()
 print(options)
 
-process = cms.Process("G4PrintGeometry",Run3_dd4hep)
+baseName = "cmsDD4hep" + options.geometry
+geomName = "Configuration.Geometry.GeometryDD4hepExtended" + options.geometry + "Reco_cff"
+
+from Configuration.ProcessModifiers.dd4hep_cff import dd4hep
+if (options.geometry == "2018"):
```
Comment (perrotta) [inline SimG4Core/PrintGeomInfo/test/python/g4OverlapCheckDD4hep_cfg.py:30]:
```suggestion
if (options.geometry == "2017"):
```

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by bsunanda** [comment]
@civanch @srimanob @perrotta Could you please re-sign this?

**Comment by perrotta** [comment]
merge

**Pull request merged**